utils/sane-backends/Makefile

```diff
@@ -127,6 +127,9 @@ $(call Package/sane-backends/Default)
   CATEGORY:=Utilities
   DEPENDS:=+libsane
   TITLE+= (network daemon)
+  USERID=saned:saned
+endef
```

Comment:
On 16-03-24 11:53 PM, Luiz Angelo Daros de Luca wrote:
> In utils/sane-backends/Makefile
> <https://github.com/openwrt/packages/pull/2543#discussion_r57416917>:
>
>> @@ -127,6 +127,9 @@ $(call Package/sane-backends/Default)
>>    CATEGORY:=Utilities
>>    DEPENDS:=+libsane
>>    TITLE+= (network daemon)
>> +  USERID=saned:saned
>> +endef
>
> does it compile? Two endef?

Hmmm...maybe my patch got mangled when I was pulling out from my working 
tree.  It compiled the last build I did, but let me verify I pull the 
patch out right.

Regards,

Daniel